tests/layer_tests/tensorflow_tests/test_tf_ScatterND.py

```diff
@@ -0,0 +1,103 @@
+
+            scatter_nd = tf.scatter_nd(tf_indices, tf_updates, tf.shape(x), name="Operation")
+            rs = tf.add(x, scatter_nd)
+            tf.nn.relu(rs)
+            tf.compat.v1.global_variables_initializer()
+
+            tf_net = sess.graph_def
+
+        ref_net = None
+        indicies_np = int64_array(indices)
+        updates_np = float32_array(updates)
+        if check_ir_version(10, None, ir_version) and not use_new_frontend:
+            const_for_layer_tests = lambda name, value, shape, shape1: {
+                **{name + '_dd': {'kind': 'data', 'value': value, 'shape': shape1}},
+                **{name: {'kind': 'op', 'type': 'Const'}},
+                **shaped_data(name + '_d', shape)}
+
+            connect_const_for_layer_tests = lambda first_tensor_name, second_tensor_name: [
+                *connect_front(first_tensor_name + '_dd', first_tensor_name),
+                *connect(first_tensor_name, second_tensor_name)]
+
+            nodes_attributes = {
+                **regular_op_with_shaped_data('input', x_shape, {'type': 'Parameter'}),
+                **const_for_layer_tests('indices', indicies_np, indicies_np.shape, indicies_np.shape),
+                **const_for_layer_tests('updates', updates_np, updates_np.shape, updates_np.shape),
+                **const_for_layer_tests('zero_tensor', float32_array(0.0), int64_array([]), int64_array([])),
+                **regular_op_with_shaped_data('broadcast', x_shape, {'type': 'Broadcast'}),
+                **regular_op_with_shaped_data('shapeof', x_shape, {'type': 'Shapeof'}),
+                **regular_op_with_shaped_data('scatter_nd', x_shape, {'type': 'ScatterNDUpdate'}),
+                **regular_op_with_shaped_data('relu', x_shape, {'type': 'ReLU'}),
+                **regular_op_with_shaped_data('sum', x_shape, {'type': 'Add'}),
+                **regular_op_with_shaped_data('result', x_shape, {'type': 'Result'}),
+
+            }
+
+            ref_net = build_graph(nodes_attributes,
+                                  [*connect('input', '0:shapeof'),
+                                   *connect_const_for_layer_tests('zero_tensor', '0:broadcast'),
+                                   *connect('shapeof', '1:broadcast'),
+                                   *connect('broadcast', '0:scatter_nd'),
+                                   *connect_const_for_layer_tests('indices', '1:scatter_nd'),
+                                   *connect_const_for_layer_tests('updates', '2:scatter_nd'),
+                                   *connect('input', '0:sum'),
+                                   *connect('scatter_nd', '1:sum'),
+                                   *connect('sum', '0:relu'),
+                                   *connect('relu', '0:result')])
+
+        return tf_net, ref_net
+
+    test_data = [pytest.param(
+        dict(x_shape=[8], indices=[[4], [3], [1], [7]], updates=[9.0, 10.0, 11.0, 12.0]),
+        marks=pytest.mark.precommit),
```

Comment:
please add to pre-commit two more complicated cases.

Reply:
done